src/test/groovy/com/avioconsulting/mule/linter/rule/configuration/UnusedFlowRuleTest.groovy

```diff
@@ -0,0 +1,74 @@
+package com.avioconsulting.mule.linter.rule.configuration
+
+import com.avioconsulting.mule.linter.TestApplication
+import com.avioconsulting.mule.linter.model.Application
+import com.avioconsulting.mule.linter.model.CaseNaming
+import com.avioconsulting.mule.linter.model.Rule
+import com.avioconsulting.mule.linter.model.RuleViolation
+import spock.lang.Specification
+
+
```

Comment:
On tests add this suppress on the class.... so you don't need to do it on each method
@SuppressWarnings(['MethodName', 'MethodReturnTypeRequired', 'StaticFieldsBeforeInstanceFields'])